mqtt-level-store.js

```diff
@@ -146,10 +146,42 @@ function Manager (path, options) {
 Manager.single = Store
 
 Manager.prototype.close = function (done) {
-  this.incoming.close()
-  this.outgoing.close()
-  this._sublevel.close()
-  this._level.close(done)
+  var incomingCloseCalled = false
+  var outgoingCloseCalled = false
+  var subLevelCloseCalled = false
+  var levelCloseCalled = false
+  var errors = {}
+
+  var tryAllClosed = function () {
```

Comment:
Can you do just use a named function instead?

Reply:
I changed it.